src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+@import "../styles/utils/mixins.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-weight: 400;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 0;
+    margin-bottom: 40px;
+  }
+
+  &__product-image {
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main-accent;
+
+    &:hover {
+      color: $color-card-title-hover;
```

Comment:
What about transition

Reply:
And also color should be changed on card hover(not only title)
<img width="796" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/81ffdd0e-e89d-4cdb-a417-7a149c21a777">